packages/react-vapor/src/components/toast/tests/Toast.spec.tsx

```diff
@@ -273,4 +273,40 @@ describe('Toasts', () => {
             expect(onCloseToast).not.toHaveBeenCalled();
         });
     });
+
+    describe('<Toast /> with download section', () => {
+        beforeEach(() => {
+            toastBasicAttributes = {
+                title: 'Preparing file for download...',
+                isDownload: true,
+            };
```

Comment:
To make the tests as clear as possible I would avoid using those props and define them directly in all the tests